specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -160,6 +160,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/checkQuotaAvailability": {
+      "post": {
+        "tags": [
+          "NetApp Resource"
+        ],
+        "summary": "Check quota availability",
+        "description": "Check if a quota is available.",
+        "operationId": "NetAppResource_CheckQuotaAvailability",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Quota availability request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/QuotaAvailabilityRequest"
+            },
+            "x-ms-client-flatten": true
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/location"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceNameAvailability"
```

Comment:
do you want to rename the model name since this API checks the quotaAvailability and not name ?

Reply:
renamed